vllm/lora/models.py

```diff
@@ -320,11 +297,22 @@ def __init__(
         self.indices_len = [None] * 4
 
         self.model: nn.Module = model
-        self.lora_target_modules: List[str] = ([
-            lora_target_modules
-        ] if isinstance(lora_target_modules, str) else lora_target_modules)
-        self.lora_target_modules = copy.deepcopy(lora_target_modules)
-        self.packed_modules_mapping = copy.deepcopy(packed_modules_mapping)
+        # allow overriding the target modules and mapping with initialization
+        if support_lora_modules:
+            self.support_lora_modules: List[str] = (
+                [support_lora_modules] if isinstance(
```

Comment:
We don't need the `isinstance` part any more now that only lists are supported, right?